README.md

```diff
@@ -76,18 +79,16 @@ of an extra kwarg "attachment":
 import subprocess
 from robotframework_reportportal import logger
 
-class MyLibrary(object):
-
-    def log_free_memory(self):
-        logger.debug("Collecting free memory statistics!")
-        logger.debug(
-            "Memory consumption report",
-            attachment={
-                "name": "free_memory.txt",
-                "data": subprocess.check_output("free -h".split()),
-                "mime": "application/octet-stream",
-            },
-        )
+def log_free_memory():
+    logger.debug("Collecting free memory statistics!")
+    logger.debug(
+        "Memory consumption report",
+        attachment={
+            "name": "free_memory.txt",
+            "data": subprocess.check_output("free -h".split()),
+            "mime": "application/octet-stream",
```

Comment:
Why use binary type? I think `text/plain` would be more suitable.

Reply:
It's been there for ages. That is just an example.

Reply:
```suggestion
            "mime": "text/plain",
```
Let's start fixing the things.